themes-default/slim/src/store/modules/defaults.js

```diff
@@ -19,30 +19,39 @@ const state = {
             sports: null,
             subtitlesEnabled: null
         },
-        countries: null,
-        country_codes: null, // eslint-disable-line camelcase
-        genres: null,
+        countries: [],
+        countryCodes: [],
+        genres: [],
         id: {
             tvdb: null,
+            tmdb: null,
+            tvmaze: null,
+            imdb: null,
             slug: null
         },
         indexer: null,
         language: null,
         network: null,
         nextAirDate: null,
         plot: null,
-        rating: {
-            imdb: {
-                rating: null,
-                votes: null
-            }
+        imdbInfo: {
```

Comment:
Maybe this would be better as `indexer: { imdb, trakt, ... }` since Trakt, etc. can also have different ratings, etc. vs IMDB. Personally I'd prefer Trakt data over `IMDB`.

Reply:
`indexer` is already in use as the indexer name, plus, it's not really indicative of the data it contains
:thinking: 

P.S.
Trakt users *always* rate the movies with low scores even before they come out, so annoying.

Reply:
Im confused. In the other discussion you mentioned that having a multi indexer structure would get messy. I get that. But then why add imdInfo here?

Then just decouple it into a dedicated call? I mean next we could finish the imdb indexer. Wouldn't we need to change this eventually?

Reply:
To have it clean we should only return tvdb rating for a tvdb show?

Reply:
> Im confused. In the other discussion you mentioned that having a multi indexer structure would get messy. I get that. But then why add imdbInfo here?

@p0psicles 
Alright maybe I didn't fully get the point you were trying to make in the other discussion.
Maybe we were talking about the same thing, and just had different structures in mind?
Did you mean calling `GET /api/v2/series/tvdb12345` would return:
```json5
{
	"tvdb": {
		/* data from tvdb */
	},
	"imdb": {
		/* data from imdb */
	},
	"tmdb": {
		/* data from tmdb */
	}
}
```
Because that's what I thought was messy.

> Then just decouple it into a dedicated call?

I hate that part of the UI change,
because I thought everything we need would be within the reach of **one** API call.
I thought everything will be like the `/config` endpoint where you can get different sections of data, or all of them together.

> I mean next we could finish the imdb indexer. Wouldn't we need to change this eventually?

<sub>Personally, I don't see how imdb can be a better indexer than any of the other 3.</sub>
I mean, yes, `imdbInfo` would be useless for the shows that would use imdb as an indexer...

> To have it clean we should only return tvdb rating for a tvdb show?

It's this whole "consolidating information from different sources" thing that has me confused.
We serve the data from the show's main indexer,
and then some of the information is collected from other indexers (TMDB - countries/country codes),
and some of it from "external" indexers (IMDB - rating, votes, ...)
It was fine for using in mako because you can always access all the information stored in the Series object, but now, you have to decide where it's going to go and how to structure all the possible info,
when sometimes the info comes **directly** from the main indexer
and sometimes it doesn't...
:confused: :confounded: